styles/layout/AppNav.module.scss

```diff
@@ -0,0 +1,27 @@
+@use "../scss/utils/variable" as *;
+
+.app_nav {
+  position: sticky;
+  top: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 23px;
+  height: $navHeight;
+  line-height: $navHeight;
+  z-index: 10;
+  background-color: $primaryColor;
+  color: rgb(255, 255, 255);
```

Comment:
Could this one be replaced with:
$primaryColor: #de473a;
$secondaryColor: #fefdfc;
$accentColor: #ff7383;
$textPrimaryColor: #333333;
$textSecondaryColor: #555555; ?